lib/WP_Auth0_DBManager.php

```diff
@@ -45,6 +45,7 @@ public function install_db( $version_to_install = null, $app_token = '' ) {
 		$sso             = $options->get( 'sso' );
```

Comment:
Just comments in this file ... I was researching the DB version and wanted to persist what I found.